pkg/common/config/config.go

```diff
@@ -394,6 +394,14 @@ func FromEnvToGC(ctx context.Context, cfg *Config) error {
 	if cfg.GC.Port == "" {
 		cfg.GC.Port = DefaultGCPort
 	}
+	// Set default fss configmap name if SV FSS configmap info is not available in GC Config
+	if cfg.FeatureStatesConfig.Name == "" {
```

Comment:
This does not belong in this method. I think this should be moved to `ReadGCConfig`. Same comment applies for cfg.GC.Port assignment.

Reply:
Agreed!

Reply:
Should it not be in `GetGCconfig` instead of `ReadGCConfig`? There is a possibility that these values can be empty if a conf file is not given in GC and only env vars are used.

Reply:
If ENV vars are used, the current logic still loads default values when conf is not given